Assets/MixedRealityToolkit/Interfaces/EventSystem/IMixedRealityEventSystem.cs

```diff
@@ -16,9 +16,9 @@ public interface IMixedRealityEventSystem : IMixedRealityService
         /// <summary>
         /// List of event listeners that are registered to this Event System.
         /// </summary>
-        // Marking EventListeners as obsolete triggers a lot of warnings in MRTK code base.
-        // This should be commented out once follow up task #4847 is complete.
-        //[Obsolete("EventListeners is replaced by EventHandlersByType and will be removed in a future release.")]*/
+        /// <remarks>
+        /// This collection is obsolete is replaced by handler-based storage and will be removed in a future release.
```

Comment:
> handler-based storag [](start = 54, length = 21)

Nit: It would be helpful to point to the specific version that is recommended instead (i.e, RegisterHandler). I think that the autocomplete tooltip should show that, so hopefully it'll be one less click/step to find the replacement.